Packs/CheckpointFirewall/Integrations/CheckPointFirewallV2/README.md

```diff
@@ -1571,6 +1571,200 @@ Shows the entire Access Rules layer. This layer is divided into sections. An Acc
+                    "name": "Any",
+                    "type": "CpmiAnyObject",
+                    "domain": {
+                        "uid": "a0bbbc99-adef-4ef8-bb6d-defdefdefdef",
+                        "name": "Check Point Data",
+                        "domain-type": "data domain"
+                    }
+                }],
+                "action": {
+                    "uid": "6c488338-8eec-4103-ad21-cd461ac2c472",
+                    "name": "Accept",
+                    "type": "RulebaseAction",
+                    "domain": {
+                        "uid": "a0bbbc99-adef-4ef8-bb6d-defdefdefdef",
+                        "name": "Check Point Data",
+                        "domain-type": "data domain"
+                    }
+                },
+                "action-settings": {
+                    "enable-identity-captive-portal": "False"
+                },
+                "content": [{
+                    "uid": "97aeb369-9aea-11d5-bd16-0090272ccb30",
+                    "name": "Any",
+                    "type": "CpmiAnyObject",
+                    "domain": {
+                        "uid": "a0bbbc99-adef-4ef8-bb6d-defdefdefdef",
+                        "name": "Check Point Data",
+                        "domain-type": "data domain"
+                    }
+                }],
+                "content-negate": "False",
+                "content-direction": "any",
+                "time": [{
+                    "uid": "97aeb369-9aea-11d5-bd16-0090272ccb30",
+                    "name": "Any",
+                    "type": "CpmiAnyObject",
+                    "domain": {
+                        "uid": "a0bbbc99-adef-4ef8-bb6d-defdefdefdef",
+                        "name": "Check Point Data",
+                        "domain-type": "data domain"
+                    }
+                }],
+                "custom-fields": {
+                    "field-1": "",
+                    "field-2": "",
+                    "field-3": ""
+                },
+                "track": {
+                    "type": {
+                        "uid": "598ead32-aa42-4615-90ed-f51a5928d41d",
+                        "name": "Log",
+                        "type": "Track",
+                        "domain": {
+                            "uid": "a0bbbc99-adef-4ef8-bb6d-defdefdefdef",
+                            "name": "Check Point Data",
+                            "domain-type": "data domain"
+                        }
+                    },
+                    "per-session": "False",
+                    "per-connection": "True",
+                    "accounting": "False",
+                    "alert": "none"
+                },
+                "layer": "7f7308f0-7540-4631-832c-503de7b27c3d"
+            }
+        ]
+    }
+}
+```
+
+#### Human Readable Output
+
+>### CheckPoint data for access rule 48604d56-b4ae-45e4-8757-29510f693a33 in layer 7f7308f0-7540-4631-832c-503de7b27c3d:
```

Comment:
This actually renders believe it or not
<img width="1074" alt="Screenshot 2025-03-19 at 9 17 20" src="https://github.com/user-attachments/assets/e19bf349-a7f2-4b0b-a989-c92409f966d6" />